pkgs/misc/tmux-plugins/default.nix

```diff
@@ -548,6 +548,39 @@ in rec {
     '';
   };
 
+  session-wizard = mkTmuxPlugin rec {
+    pluginName = "session-wizard";
+    # rtpFilePath = "session-wizard.tmux";
```

Comment:
Dead code.

Reply:
Sorry, it's not dead code, it's my mistake. I don't know what is normal flow so I amended it and push forced.

Reply:
Amending and pushing is fine, don't worry.